src/controllers/commands/builtin/chatterino/Debugging.cpp

```diff
@@ -0,0 +1,44 @@
+#include "controllers/commands/builtin/chatterino/Debugging.hpp"
+
+#include "common/Channel.hpp"
+#include "controllers/commands/CommandContext.hpp"
+#include "messages/MessageBuilder.hpp"
+
+#include <QLoggingCategory>
+#include <QString>
+
+namespace chatterino::commands {
+
+QString setLoggingRules(const CommandContext &ctx)
+{
+    if (ctx.words.size() < 2)
+    {
+        ctx.channel->addMessage(makeSystemMessage(
+            "Usage: /c2:loggingrules <rules...>. To enable debug logging for "
+            "all "
+            "categories from chatterino, use 'chatterino.*.debug=true'. For "
+            "the format on the rules, see "
+            "https://doc.qt.io/qt-6/"
+            "qloggingcategory.html#configuring-categories"));
+        return {};
```

Comment:
We return `""` at the end of the function - I'm fine with either but we should use the same in both places